nncf/onnx/quantization/quantizer_parameters.py

```diff
@@ -68,6 +68,8 @@ def convert_fq_params_to_onnx_params(
     # ONNX demands parameters to be a scalar or 1-D Tensor.
     scale = np.squeeze(scale)
     zero_point = np.squeeze(zero_point)
+    # ONNX axis parameter format specification.
+    axis = None if not axis else axis[0]
```

Comment:
Just to make sure: not axis means axis == 0 or axis is None?

Reply:
axis here is a Tuple. This checks whether axis has elements or not.